### PR TITLE
README md package create-frontity

### PR DESCRIPTION
**Pull request by juanmaguitar**
**What**:

README.md's create-frontity

**Why**:

Proper info in NPM

**How**:

Creating the README.md's using on the proper folder

```
npx readme-md-generator -p ../../.github/README_TEMPLATE/package-readme.md -y
```

And then adding custom info on each README.md

**Tasks**:

**Unrelated Tasks**

- Code
- TSDocs
- TypeScript
- Unit tests
- End to end tests
- TypeScript tests
- Update starter themes
- Update other packages
- Link to PR in [Documentation](https://github.com/frontity/gitbook-docs/)
- Community discussions
- Changeset (with link to its [Feature Discussion](https://community.frontity.org/c/33) if it exists)

**Additional Comments**

### CODE REVIEW

File: .changeset/nervous-schools-teach.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"create-frontity": minor
```
Comment (luisherranz) [inline .changeset/nervous-schools-teach.md]: Please change this to a `patch`.

File: .changeset/nervous-schools-teach.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+README.md full description
```
Comment (luisherranz) [inline .changeset/nervous-schools-teach.md]: Please use proper English phrases for the changelog, ending them with a period.

**Review by luisherranz — CHANGES_REQUESTED** [review]

File: packages/create-frontity/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  * [» Get involved 🤗](#%C2%BB-get-involved-%F0%9F%A4%97)
+
+<!-- tocstop -->
+
+## Install
+
+```sh
+npm i create-frontity
+```
+
```
Comment (luisherranz) [inline packages/create-frontity/README.md]: This package doesn't need to be installed. It's meant to be used only in conjunction with the `npm init` command, like this:

```bash
> npm init frontity
```

**Review by luisherranz — CHANGES_REQUESTED** [review]

**Review by luisherranz — APPROVED** [review]
👍

**Comment by michalczaplinski** [comment]
Hey @juanmaguitar I've merged this one and the other remaining PRs with the READMEs.

Just as a note for the future - we have a convention that it's normally up to the PR author to merge the PR after an approval. I don't think that it's explicitly documented so you couldn't know that 🙂

**Pull request merged**